openpype/hosts/houdini/plugins/publish/validate_workfile_paths.py

```diff
@@ -24,6 +24,16 @@ class ValidateWorkfilePaths(
     node_types = ["file", "alembic"]
     prohibited_vars = ["$HIP", "$JOB"]
 
+    @classmethod
+    def apply_settings(cls, project_settings, system_settings):
+        """Apply project settings to creator"""
+        settings = (
+            project_settings["houdini"]["publish"]["ValidateWorkfilePaths"]
+        )
+
+        cls.node_types = settings.get("node_types", cls.node_types)
+        cls.prohibited_vars = settings.get("prohibited_vars", cls.prohibited_vars)
```

Comment:
Isn't this basically the default behavior already @iLLiCiTiT ? Or do we need to implement this anyway for AYON?

Reply:
This is indeed defaul behavior.